### PR TITLE
Minor updates to download and daily build information

### PR DESCRIPTION
**Pull request by hamishwillee**
These changes are intended to make the most common download and daily build activities easier. There are still some open issues before this can/should be merged.

### CODE REVIEW

File: en/releases/daily_builds.md [hunk 1]
```diff
@@ -1,20 +1,28 @@
...
-Daily Builds of QGroundControl are also available for use. These builds have less testing than Stable builds but will have the absolute latest set of [new features](../releases/daily_build_new_features.md) available. Use at your own risk!
+Daily Builds of *QGroundControl* have the absolute latest set of [new features](../releases/daily_build_new_features.md). 
+
+> **Warning** Daily Builds are less tested than stable builds. Use at your own risk!
+
+These can be downloaded from the links below:
 
 * [Windows](https://s3-us-west-2.amazonaws.com/qgroundcontrol/builds/master/QGroundControl-installer.exe)
 * [OS X](https://s3-us-west-2.amazonaws.com/qgroundcontrol/builds/master/QGroundControl.dmg)
 * Linux
```
Comment (hamishwillee) [inline en/releases/daily_builds.md]: @DonLakeFlyer Would I be correct that for Linux you either install the AppImage, OR you install the Compressed Archive AND the Qt libraries (ie you only need the additional packages IF you use the archive)?

File: en/releases/daily_builds.md [hunk 2]
```diff
@@ -1,20 +1,28 @@
...
   * [AppImage](https://s3-us-west-2.amazonaws.com/qgroundcontrol/builds/master/QGroundControl.AppImage)
   * [Compressed Archive](https://s3-us-west-2.amazonaws.com/qgroundcontrol/builds/master/QGroundControl.tar.bz2)
 * [Android](https://play.google.com/store/apps/details?id=org.mavlink.qgroundcontrolbeta&rdid=org.mavlink.qgroundcontrolbeta)
```
Comment (hamishwillee) [inline en/releases/daily_builds.md]: @DonLakeFlyer This link does not work. Is there a correct link, or some other way to get the Android beta access?

**Comment by DonLakeFlyer** [comment]
The android daily builds are currently suspended which is why the link isn't showing up. I think this is the privacy policy thing. I've asked Daniel what is left to do there.

**Comment by hamishwillee** [comment]
OK. This can go out with the broken link (probably you just need to add link to the [privacy policy](https://docs.qgroundcontrol.com/en/releases/google_play_privacy_policy.html) and let google know).

@DonLakeFlyer What about the other question?: "Would I be correct that for Linux you either install the AppImage, OR you install the Compressed Archive AND the Qt libraries (ie you only need the additional packages IF you use the archive)?"

**Comment by DonLakeFlyer** [comment]
> you only need the additional packages IF you use the archive

Correct

File: en/releases/daily_builds.md (hunk 2 above)
Comment (hamishwillee) [inline en/releases/daily_builds.md]: @dagar Any progress on sorting out Android beta access?

**Comment by hamishwillee** [comment]
Hi @DonLakeFlyer , I'd like to simplify the Download and Install section by merging the download, versioning and "how to install" info where possible. There are two versions. Would appreciate your opinion.

Version 1/2 below - here we have table for most bits. There is a link down below for linux instructions. I like a section for linux instructions because we can refer to them in daily build instructions too.

----

# Download and Install

<table> 
<thead>
<tr> 
 <th width="25%">OS</th> 
 <th>Download</th>
 <th>Install</th> 
 </tr> 
</thead>
<tbody>
<tr> <td>Windows<br>(Windows Vista or later) </td> <td><a href="https://s3-us-west-2.amazonaws.com/qgroundcontrol/latest/QGroundControl-installer.exe">QGroundControl-installer.exe</a></td>  <td>Double-click the executable to launch the installer.</td> </tr> 
<tr> <td>Mac OS X<br>(10.8 or above)</td> <td><a href="https://s3-us-west-2.amazonaws.com/qgroundcontrol/latest/QGroundControl.dmg">QGroundControl.dmg</a></td>  <td>Double-click the .dmg file to mount it, then drag the QGroundControl application to your Application folder.</td> </tr> 
<tr> <td rowspan="2">Ubuntu Linux<br>(14.04 LTS or above)</td> <td><a href="https://s3-us-west-2.amazonaws.com/qgroundcontrol/latest/QGroundControl.AppImage">QGroundControl.AppImage</a></td> <td>See instructions below [link]</td> </tr> 
<tr> <td><a href="">QGroundControl.tar.bz2</a></td> <td>See instructions below <LINK>.</td>  </tr> 
<tr> <td>Android<br>(Android 5.1 or above)</td> <td><a href="https://play.google.com/store/apps/details?id=org.mavlink.qgroundcontrol">Google Play Store</a> </td> <td>Click link and follow instructions on Google Play.</td>  </tr> 
<tr><td>iOS<br>(iOS 8.0 or above)</td><td>-</td><td>Beta release - follow <a href="../releases/daily_builds.md">iOS daily build instructions</a></td></tr>
<tbody>
</table>


## Installing on Ubuntu Linux

Install QGroundControl for Ubuntu Linux 14.04 LTS or later. You can either install the AppImage **or** the compressed archive (and additional packages).

### AppImage

1. Download [QGroundControl.AppImage](https://s3-us-west-2.amazonaws.com/qgroundcontrol/latest/QGroundControl.AppImage)
1. Install using the terminal commands:
   ```sh
   chmod +x ./QGroundControl.AppImage
   ./QGroundControl.AppImage (or double click)
   ```

### Compressed Archive

1. Download [QGroundControl.tar.bz2](https://s3-us-west-2.amazonaws.com/qgroundcontrol/latest/QGroundControl.tar.bz2)
1. Extract the archive using the terminal command:
   ```sh
   tar jxf QGroundControl.tar.bz2
   cd qgroundcontrol
   ./qgroundcontrol-start.sh
   ```
1. Install additional packages as specified in the github <a class="urlextern" title="https://github.com/mavlink/qgroundcontrol" href="https://github.com/mavlink/qgroundcontrol" rel="nofollow">README</a>. You do not need to install Qt.

**Comment by hamishwillee** [comment]
Version 2/2 below. Here we just have sections for each. It is no more compact than what you have currently, but I prefer it because all the information about each platform is together. Thoughts?

--------

## Download and Install

The sections below can be used download the current stable release of *QGroundControl* ([release notes](../releases/release_notes.md)) for each platform:

## Windows

Install QGroundControl for Windows Vista or later:

1. Download [QGroundControl-installer.exe](https://s3-us-west-2.amazonaws.com/qgroundcontrol/latest/QGroundControl-installer.exe)
1. Double click the executable to launch the installer.


## Mac OS X

Install QGroundControl for Mac OS X 10.8 or later: 

1. Download [QGroundControl.dmg](https://s3-us-west-2.amazonaws.com/qgroundcontrol/latest/QGroundControl.dmg)
1. Double-click the .dmg file to mount it, then drag the QGroundControl application to your Application folder.

  
## Ubuntu Linux

Install QGroundControl for Ubuntu Linux 14.04 LTS or later. You can either install the AppImage **or** the compressed archive (and additional packages).

### AppImage

1. Download [QGroundControl.AppImage](https://s3-us-west-2.amazonaws.com/qgroundcontrol/latest/QGroundControl.AppImage)
1. Install using the terminal commands:
   ```sh
   chmod +x ./QGroundControl.AppImage
   ./QGroundControl.AppImage (or double click)
   ```

### Compressed Archive

1. Download [QGroundControl.tar.bz2](https://s3-us-west-2.amazonaws.com/qgroundcontrol/latest/QGroundControl.tar.bz2)
1. Extract the archive using the terminal command:
   ```sh
   tar jxf QGroundControl.tar.bz2
   cd qgroundcontrol
   ./qgroundcontrol-start.sh
   ```
1. Install additional packages as specified in the github <a class="urlextern" title="https://github.com/mavlink/qgroundcontrol" href="https://github.com/mavlink/qgroundcontrol" rel="nofollow">README</a>. You do not need to install Qt.
    

  
## Android

Install QGroundControl for Android 5.1 or later:

1. Open the Google Play Store [QGroundControl link](https://play.google.com/store/apps/details?id=org.mavlink.qgroundcontrol)
1. Follow the installation instructions.


## iOS (Beta)

You can install QGroundControl on iOS 8.0 or later. QGroundControl for iOS is in beta. 

1. Follow the instructions for accessing the [iOS daily build](../releases/daily_builds.md)

File: en/releases/daily_builds.md (hunk 2 above)
Comment (dagar) [inline en/releases/daily_builds.md]: I contacted google support...so no. Some automated system flagged it as impersonating the main QGroundControl, even though they're under the same account.
I'll try to follow up.

File: en/releases/daily_builds.md (hunk 1 above)
Comment (DonLakeFlyer) [inline en/releases/daily_builds.md]: That is correct

**Comment by hamishwillee** [comment]
@DonLakeFlyer OK, I've merged a slightly improved "version2".
This also makes some changes to the daily build page. Specifically the Android section now has link to track the locked out build https://github.com/mavlink/qgroundcontrol/issues/5162 . I also have a note in the same place on the "opt out" for the old beta. At some point that note can probably go, since everyone will have opted out. I'm happier with this.

**Comment by hamishwillee** [comment]
@jeyong FYI, This update changes the format of pages that would probably be better also being changed in the Korean translation. There are a number of pages, but the most important are download_and_install.md and /releases/daily_builds.md

**Comment by jeyong** [comment]
@hamishwillee Thanks for notice. I updated changes.

**Pull request merged**